frontends/p4/tableKeyNames.h

```diff
@@ -23,13 +23,6 @@ limitations under the License.
 
 namespace P4 {
 
-/**
```

Comment:
Technically this comment is correct, this is the pass that adds the annotations.
The PassManager below just calls this one.